src/App.js

```diff
@@ -14,11 +15,110 @@ const goodsFromServer = [
   'Garlic',
 ];
 
-const App = () => (
-  <div className="App">
-    <h1>Goods</h1>
-    {goodsFromServer.length}
-  </div>
-);
+class App extends React.Component {
+  state = {
+    goods: [],
+    hideButton: false,
```

Comment:
```suggestion
    isHiddenButton: false,
```
або щось на кшталт isClicked isStarted